docarray/documents/helper.py

```diff
@@ -118,3 +118,24 @@ class MyAudio(TypedDict):
     doc = create_model_from_typeddict(typeddict_cls, **kwargs)
 
     return doc
+
+def create_from_dict(
+    data: Dict[str,Any],  # type: ignore
```

Comment:
why type ignore here ?

Reply:
I did not add any type check here as I thought the dictionary could contain any type as values so I used `Any` here. Should I do something else here?